src/cephadm/cephadm

```diff
@@ -3870,7 +3880,7 @@ def list_daemons(detail=True, legacy_dir=None):
             elif is_fsid(i):
                 fsid = str(i)  # convince mypy that fsid is a str here
                 for j in os.listdir(os.path.join(data_dir, i)):
-                    if '.' in j:
+                    if '.' in j and os.path.isdir(os.path.join(data_dir, fsid, j)):
```

Comment:
do you still need this line?

Reply:
no now that i switched back to /etc/ceph its not needed.   i do think its a good check to keep though. any file put in /var/lib/ceph/fsid that has a . in it will be treated as an orphan daemon like the screen shot i had above.